tests/console/system_prepare.pm

```diff
@@ -87,6 +87,8 @@ sub run {
             services::registered_addons::full_registered_check;
         }
     }
+
+    script_run 'rpm -q systemd-coredump || zypper -n in systemd-coredump' if get_var('COLLECT_COREDUMPS');
```

Comment:
I understand your need but doing it like this. But I suggest to still use `assert_script_run` and handle the "… or not" inside the shell call. Otherwise you will sooner or later end up with a "red failed box" which confuses test reviewers and leads to either alarm fatigue or red herrings.

Reply:
updated - verification: http://kazhua.qa.suse.de/tests/133#step/system_prepare/10